src/ukify/test/test_ukify.py

```diff
@@ -219,12 +219,15 @@ def test_config_priority(tmp_path):
         DeviceTree = some/path2
         Splash = some/path3
         Uname = 1.2.3
-        EFIArch=arm
+        EFIArch = arm
```

Comment:
Not that it matters greatly, but the style is *inconsistent* here on purpose: we want to test various whitespace arrangements. But in fact the variant without spaces is already used in an example earlier, so I it's fine to add the spaces here.